xbmc/storage/MediaManager.cpp

```diff
@@ -716,29 +716,26 @@ CMediaManager::DiscInfo CMediaManager::GetDiscInfo(const std::string& mediaPath)
   if (mediaPath.empty())
     return info;
 
-  // Try finding VIDEO_TS/VIDEO_TS.IFO - this indicates a DVD disc is inserted
+  // Check for DVD discs
   std::string pathVideoTS = URIUtils::AddFileToFolder(mediaPath, "VIDEO_TS");
-  if (CFile::Exists(URIUtils::AddFileToFolder(pathVideoTS, "VIDEO_TS.IFO")))
```

Comment:
would there ever be a case that a DVD wouldn't have this?

Reply:
There is at least a case where the video_ts.ifo is in the root folder and not under video_ts: https://github.com/xbmc/xbmc/blob/d6a4bc3f030d5e57e8e706ff5c759fa3e812eca0/xbmc/FileItem.cpp#L1829-L1832
Either way, this is used for dvd stacking / dvd folder storage on disk and not exactly to request playback from the optical drive (so a different code path). For this PR/fix if we manage to open the dvd with libdvdnav/read we can safely assume the item is a DVD.